website/pages/api-docs/connect/intentions.mdx

```diff
@@ -60,8 +198,9 @@ The table below shows this endpoint's support for
 - `DestinationNS` `(string: "")` <EnterpriseAlert inline /> - The namespace for the
   `DestinationName` parameter.
 
-- `SourceType` `(string: <required>)` - The type for the `SourceName` value.
-  This can be only "consul" today to represent a Consul service.
+- `SourceType` `(string: "")` - The type for the `SourceName` value.
+  This can be only `"consul"` today to represent a Consul service. If not
+  provided, this will be defaulted to `"consul"`.
```

Comment:
Resolved this the other way around (keeping the quotes instead of the backticks). This aligned with other example strings already in the docs like `"allow"` or `"deny"`.